pkg/extensions/sync/sync_test.go

```diff
@@ -1144,25 +1089,18 @@ func TestSyncNoImagesByRegex(t *testing.T) {
 			dc.Shutdown()
 		}()
 
-		Convey("Test sync on POST request on /sync", func() {
-			resp, err := destClient.R().Post(destBaseURL + "/sync")
-			So(err, ShouldBeNil)
-			So(resp, ShouldNotBeNil)
-			So(resp.StatusCode(), ShouldEqual, 200)
-
-			resp, err = destClient.R().Get(destBaseURL + "/v2/_catalog")
-			So(err, ShouldBeNil)
-			So(resp, ShouldNotBeEmpty)
-			So(resp.StatusCode(), ShouldEqual, 200)
+		resp, err := destClient.R().Get(destBaseURL + "/v2/_catalog")
```

Comment:
I think to simulate better  the test intention we need to add a delay (time.Sleep) before checking that downstream server contains no images. Ideally the upstream server should contain some images to make sure downstream does not synced them.